src/intervals/conversion_promotion.jl

```diff
@@ -2,14 +2,12 @@
 
 ## Conversion and promotion
 
-## Default conversion is to Interval{Float64}
+## Default conversion to Interval, corresponds to Interval{Float64}
 convert{T<:Real}(::Type{Interval}, x::T) = make_interval(Float64, x)
```

Comment:
I think this is the most reasonable thing, in the sense that usually we use `Float64`; in addition, this is along the same lines as `∅` defaults to `emptyinterval(Float64)`.